src/infra/postgres/postgres.go

```diff
@@ -16,4 +16,10 @@ type Client interface {
 	DeletePK(ctx context.Context, model ...interface{}) error
 	ForceDeletePK(ctx context.Context, model ...interface{}) error
 	RunInTransaction(ctx context.Context, persist func(client Client) error) error
+
+	SelectMany(ctx context.Context, model interface{}, dst interface{}, condition string, params ...interface{}) error
+
+	Ping(ctx context.Context) error
+	CreateTable(ctx context.Context, model interface{}) error
+	DropTable(ctx context.Context, model interface{}) error
```

Comment:
Same comment about this should not appear here as it's not used by the app.

Reply:
We decided to keep it in the review finally until we use real migrations even for tests.

Reply:
I cannot see reason for this either, sorry :)

Reply:
Removed it as it is not used by anything anymore.